src/PrestaShopBundle/Resources/views/Admin/Common/Grid/Blocks/Table/filters_row.html.twig

```diff
@@ -23,7 +23,7 @@
  * International Registered Trademark & Property of PrestaShop SA
  *#}
 
-<tr class="column-filters {% if 0 == grid.data.records_total and grid.filters is empty and grid.actions.bulk|length < 0 %}d-none{% endif %}">
+<tr class="column-filters{% if grid.attributes.is_empty_state %} d-none{% endif %}">
```

Comment:
@matks except for `and grid.actions.bulk|length < 0` which was also removed in the @Progi1984's [PR](https://github.com/PrestaShop/PrestaShop/pull/19606/files#diff-d3df3c524df921323ec17cedfe38888bL26) for the same reason, it's the exact same condition in twig that [the condition defining if `is_empty_state` is true or not](https://github.com/PrestaShop/PrestaShop/blob/develop/src/Core/Grid/Presenter/GridPresenter.php#L89) and it's already used for [tables header row](https://github.com/PrestaShop/PrestaShop/blob/develop/src/PrestaShopBundle/Resources/views/Admin/Common/Grid/Blocks/Table/headers_row.html.twig#L26)

Reply:
OK I understand it better. I was initially confused by "empty state" because I thought it was about these images that appear instead of an empty listing.

So it's something different here. If I track correctly your train of thoughts, "a grid is in an empty state when 1. it has no records 2. there is no registered filter 3. this is not a search result"

The implementation seems valid however what I am wary of in this PR is that `search_request` parameter is being used in GridPresenter (= so it's used by ALL grids) but is only being used into CMS Pages grid. So we have a global parameter, a global property that can be used for all the grids (twenty ? thirty ?) into PrestaShop ... but is only being used for CMS Pages Grid. And then only CMS Pages Grid benefits from this (undocumented) feature.

It would be different if we were able to make sure that `is_search_request` filter is computed automatically, for any grid. That would be awesome then 😄 

Wdyt ?

Reply:
No no, your were right, it's about the placeholder images when the list is empty.

Actually, it was firstly used by the Categories page and I just reused this method but you're right, it would be better to have it automatically computed and available everywhere :)

Reply:
@matks I've finally managed to do it differently. Seems cleaner to me but lmk what you think 🙂